### PR TITLE
[tflchef] Support int8 dtype

### PR DESCRIPTION
**Pull request by jinevening**
This supports int8 dtype in tflchef.

ONE-DCO-1.0-Signed-off-by: Hyukjin Jeong <hj1.jeong@samsung.com>

---
Related to: https://github.com/Samsung/ONE/issues/11109
Draft PR: https://github.com/Samsung/ONE/pull/11155

### CODE REVIEW

**Review by hseok-oh — APPROVED** [review]
LGTM

File: compiler/tflchef/proto/tflchef.proto [hunk 1]
```diff
@@ -22,6 +22,7 @@ enum TensorType {
   STRING = 5;
   BOOL = 6;
   INT16 = 7;
+  INT8 = 8;
```
Comment (seanshpark) [inline compiler/tflchef/proto/tflchef.proto:25]: circle schema uses `INT8 = 9,` can you use same `9`  ?

Comment (jinevening) [reply compiler/tflchef/proto/tflchef.proto]: Ah, I missed the comment. I've updated the code. Thanks!

```
// This enum value corresponds to TensorType in TensorFlow Lite schema
```

**Review by seanshpark — APPROVED** [review]
LGTM thank you!

**Review by hseok-oh — APPROVED** [review]
LGTM

**Pull request merged**